### PR TITLE
completed sql-insert exercise, avallon smith

### PR DESCRIPTION
**Pull request by avallonsmith**
image
image

### CODE REVIEW

File: sql-insert/insert-actor.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+insert into "actors" ("firstName", "lastName")
+values ('Johnny', 'Depp')
```
Comment (scbowler) [inline sql-insert/insert-actor.sql:2]: 👍

File: sql-insert/insert-languages.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+insert into "languages"("name")
+values ('HTML', 'CSS', 'JavaScript')
```
Comment (scbowler) [inline sql-insert/insert-languages.sql:2]: Each row to insert needs to be in separate parentheses, so it should be:
```SQL

insert into "languages"("name")
values
  ('HTML'), 
  ('CSS'),
  ('JavaScript')
returning *;
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Really close! Just 1 update needed.

File: sql-insert/insert-languages.sql (hunk 2 above)
Comment (avallonsmith) [reply sql-insert/insert-languages.sql:2]: Shoot. I updated that in the pagila query but forgot to in the code, I'll fix it ASAP, sorry about that!

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**